test/core/componentTests.ts

```diff
@@ -403,6 +403,33 @@ describe("Component behavior", () => {
     svg.remove();
   });
 
+  it("rendering to a new svg detaches the component", () => {
```

Comment:
I think properly this test belongs somewhere else, but I think we'll leave it for now.